<?xml version="1.0"?>
<launch>
    <arg name="sim" default="false"/>
    <arg name="torqueless_mode" default="false"/>
    <arg name="only_imu" default="false"/>
    <arg name="only_pressure" default="false"/>

    <let if="$(env IS_ROBOT false)" name="taskset" value="taskset -c 0"/>
    <let unless="$(env IS_ROBOT false)" name="taskset" value=""/>

    <node pkg="bitbots_ros_control" exec="node" output="screen" launch-prefix="$(var taskset)">
      <param from="$(find-pkg-share bitbots_ros_control)/config/sahr.yaml" />
        <param name="torqueless_mode" value="$(var torqueless_mode)"/>
        <param name="only_imu" value="$(var only_imu)"/>
        <param name="only_pressure" value="$(var only_pressure)"/>
    </node>


</launch>

